app/_locales/en/messages.json

```diff
@@ -1491,6 +1491,9 @@
   "showSeedPhrase": {
     "message": "Show seed phrase"
   },
+  "showSpeedUpAnyTransactionDescription": {
+    "message": "Select this to allow resubmit transaction with any nonce."
```

Comment:
The following may be better: `Select this to allow transaction resubmission with any nonce."

Reply:
Thanks alot!

Reply:
@rachelcope For your UX consideration.

Reply:
@yellowgh0st Please go ahead and update this copy to "Select this to allow transaction resubmission with any nonce."